eng/AfterSigning.targets

```diff
@@ -17,6 +17,7 @@
       <InstallerFiles Include="$(InstallersOutputPath)**\*.tar.gz" />
       <InstallerFiles Include="$(InstallersOutputPath)**\*.wixlib" />
       <InstallerFiles Include="$(InstallersOutputPath)**\*.rpm" />
+      <InstallerFiles Remove="$(InstallersOutputPath)**\*.wixpack.zip" />
```

Comment:
Curious:
1. What adds these in the first place❔
2. Why remove them❔
3. Should this be done only when signing w/in our pipeline❔

Reply:
These aren't really anything for customers, so no need to generate sha512s. Reduces the overall number of output files signficantly. I changed the remove to an Exclude of wixpack.zips a couple lines above.